piku_dashboard/piku_client.py

```diff
@@ -79,15 +79,27 @@ def ps(appid):
 
 
 def config(appid):
-    config_lines = _piku_run(["config", appid]).split("\n")
-    config_entries = [x.split("=") for x in config_lines]
-    return {k: v for k, v in config_entries}
+    try:
+        config_lines = _piku_run(["config", appid]).split("\n")
+        config_entries = [x.split("=") for x in config_lines]
+        return {k: v for k, v in config_entries}
+    except:
+        return {}
```

Comment:
Added this because one of my app didn't have any config setup yet, and was erroring here. Rather than that I've made it so it returns an empty object

Reply:
Amazing! good catch :)

Reply:
I think this might be better if if caught a specific error, as any error (because of the try..except) will make this seem like it has no config (and result in a user potentially overwriting all configs if they don't know that)

What was the exact error you were getting?

Reply:
So I couldn't reproduce the exception I was getting, but I managed to find another problem when the config is empty!